app/pyproject.toml

```diff
@@ -260,3 +260,19 @@ test = [
     "pytest-xdist>=3.6.1",
 ]
 
+[tool.ruff]
+line-length = 120
+
+[tool.ruff.format]
+indent-style = "space"
+quote-style = "single"
+
+[tool.ruff.lint]
+select = ["E", "F", "W", "I"]
```

Comment:
Maybe add 
```
    "UP",  # pyupgrade
    "PLE",
    "PLW", # Pylint
```
other than that looks fine.

Reply:
Will add it in another PR.